src/coreclr/tools/Common/Compiler/NativeAotNameMangler.cs

```diff
@@ -290,9 +290,9 @@ private string ComputeMangledTypeName(TypeDesc type)
                     mangledName = GetMangledTypeName(((PointerType)type).ParameterType) + NestMangledName("Pointer");
                     break;
                 case TypeFlags.FunctionPointer:
-                    // TODO: need to also encode calling convention (or all modopts?)
+                    // TODO: need to also encode modopts?
```

Comment:
Given https://github.com/dotnet/runtime/pull/81122#discussion_r1090203792 I think this comment can be deleted.

I think we're going to require that after #81006 whoever is calling into this logic already normalized the function pointer type so that differences between `delegate* unmanaged[Cdecl]<void>` and `delegate* unmanaged<void>` don't exist.

Reply:
Deleted in 2nd commit.